lib/effects/sequence_effect.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:meta/meta.dart';
+
+import './effects.dart';
+import '../position.dart';
+
+class SequenceEffect extends PositionComponentEffect {
+  final List<PositionComponentEffect> effects;
+  int _currentIndex = 0;
+  PositionComponentEffect currentEffect;
+  bool _currentWasAlternating;
+  double _driftModifier = 0.0;
+
+  SequenceEffect({
+    @required this.effects,
+    isInfinite = false,
+    isAlternating = false,
+  }) : super(isInfinite, isAlternating);
```

Comment:
Add an assert here! Making sure that the effects are finite, what do you think?

Reply:
That might be a good idea, there are probably not that many usecases where you want it to get stuck.